_events/2022-All-Hands-Meeting.md

```diff
@@ -28,11 +28,11 @@ Focus areas for 2022 will include:
 - Campus services and experiences, and NSF Campus Cyberinfrastructure (CC*)
 - Impact on multi-institutional collaborations, including U.S. LHC Projects
 
-In-person participation will include a standard registration fee; online-only participation will continue to be free. Required registration for both components will open in January 2022.
+The in-person registration fee is $70; online-only participation will continue to be free. 
 
 Note that there will be an initial registration deadline of February 17. If at least 30 people register for the in-person components by that date, we can proceed with the hybrid event; otherwise, we will have to cancel the in-person components and have just the online components.
 
-For further details, visit: [https://opensciencegrid.org/all-hands/2022/](https://opensciencegrid.org/all-hands/2022/)
+To register, visit: [https://opensciencegrid.org/all-hands/2022/registration/](https://opensciencegrid.org/all-hands/2022/registration/)
```

Comment:
For the link text, how about “the AHM registration page” (removing the preceding colon)?